### ISSUE TITLE
Implement file size pattern check for merge requests(pull requests)

### ISSUE THREAD

**Issue by ar613** [christiangalsterer/stash-filehooks-plugin#23]
Version 2.4.0 introduced file name pattern check for merge requests(pull requests), but not for the file size pattern. Request for this functionality to be added.

**Comment by christiangalsterer** [comment]
Yes I'm aware of this and was already looking into it, but it is not as easy as with the file size check.
In the next 3 weeks I will have no time to look into it, so if you have some time and mood I'm happy for any PR.
